flink-streaming-java/src/main/java/org/apache/flink/streaming/api/connector/sink2/StandardSinkTopologies.java

```diff
@@ -34,7 +41,21 @@ private StandardSinkTopologies() {}
      */
     public static <CommT> void addGlobalCommitter(
```

Comment:
maybe `addGlobalCommitOperator` accordingly?

Reply:
I am not a big fan to expose the concept of an operator to users. Operator to me is more like an implementation detail.